mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -682,14 +659,14 @@ impl NftListStorageOps for SqliteNftStorage {
         token_id: BigDecimal,
     ) -> MmResult<Option<Nft>, Self::Error> {
         let table_name = chain.nft_list_table_name()?;
-        let sql = format!("SELECT * FROM {} WHERE token_address=?1 AND token_id=?2", table_name);
-        let params = [token_address, token_id.to_string()];
-        let selfi = self.clone();
-        async_blocking(move || {
-            let conn = selfi.0.lock().unwrap();
-            query_single_row(&conn, &sql, params, nft_from_row).map_to_mm(SqlError::from)
+        self.call(move |conn| {
+            let sql = format!("SELECT * FROM {} WHERE token_address=?1 AND token_id=?2", table_name);
+            let params = [token_address, token_id.to_string()];
```

Comment:
Why `token_id` is `BigDecimal`? I suppose it can have only integer values.

Reply:
Bcz `token_id` can be a super large number like in this NFT from eth:
```rust
"token_address": "0xd4416b13d2b3a9abae7acd5d6c2bbdbe25686401",
"token_id": "113962339922605038952398542560295103658755529774171739254669443553427356386098",
```
 Its better to use `BigDecimal`

Reply:
There is [BigUint](https://docs.rs/num-bigint/latest/num_bigint/struct.BigUint.html) type, which is reexported from mm2_number: `use mm2_number::BigUnit`. Please use it instead.